versioned/dynamodb/src/main/java/com/dremio/nessie/versioned/implrocksdb/RocksDBStore.java

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
I believe we talked about creating another project that had a dependency back to the Dynamo project to get access to the relevant interfaces that had not yet moved. That way you would simply have to change the project to wherever the interfaces ended up. It seems everything is created in the Dynamo project currently.

Reply:
Ah, yes. I went with different sub-package. By project do you mean Maven module, ie outside of nessie-versioned, or totally outside of Nessie, repo?

Reply:
Agreed. This should reside in package com.dremio.nessie.versioned.store.rocksdb.